src/index.ejs

```diff
@@ -9,15 +9,44 @@
     <% if (htmlWebpackPlugin.options.hasPublicPath) { %>
     <link rel="icon" type="image/png" href="/viewer/static/playcanvas-logo.png" />
     <link rel="manifest" href="/viewer/static/manifest.json">
+    <meta name="pc-url-home" content="" />
+    <meta name="pc-onetrust-key" content="<%= htmlWebpackPlugin.options.oneTrustDomainKey %>" />
```

Comment:
should we include this part without a condition  if (htmlWebpackPlugin.options.analyticsID)   ?

Reply:
Yeah because we can show it for any future cookies